### DISCUSSION TITLE
How to install Kohya_ss on linux (UBUNTU 20.04+)?

### DISCUSSION THREAD

**Discussion by runner22k** [bmaltais/kohya_ss#86] [Q&A] [answered]
Hello dear rescuer, As I am limited with 8GB VRAM, I moved to Linux. Now I want to Install kohya_ss under linux with deepspeed and test my luck. Does anyone know, how to install this on linux (UBUNTU 20.04+)? 

**NOTE:** _I switched to linux so that I can use deepspeed and get dreambooth working._

**Comment by martinobettucci** [comment]
https://github.com/P2Enjoy/kohya_ss-docker

I will update this every now and then, I can't do it more often then every week.
You're welcome to contribute.

**Comment by martinobettucci** [comment, marked as answer]
It was updated and confirmed to work on 8gb VRAM (7.9 minimum actually)

**Reply by runner22k** [reply]
please provide installation steps on your project page. People who has no experience with dockers will find it very helpful.

**Reply by martinobettucci** [reply]
Ok! Will do that during the weekend. I'm also working on a standalone version with no docker

**Reply by vivi-gomez** [reply]
What about running kohya colab script through jupyter lab? Is that possible?

**Reply by martinobettucci** [reply]
Hi @txirrindulari, it is possible to use Collab with KOYA but latest versions requres python 3.10 which is not (easily) available via Colab.
I've made a Google Collab for this but it requires a special manipulation to work: if you can't figure it out by yourself, ask me here.

https://gist.github.com/martinobettucci/4fc9f803d6e7f02d5d732a61dcf20393
